react-paw-mailmerge/src/components/FileUpload.tsx

```diff
@@ -5,6 +5,15 @@ import InfoCard from './InfoCard';
 const FileUpload: React.FC = () => {
   const [selectedFile, setSelectedFile] = useState<File | null>(null);
   const [dataHasHeader, setDataHasHeader] = useState<boolean>(true);
+  const [nullFieldOption, setNullFieldOption] = useState<string>('ignore');
+
+  const getParserOpts = (): Object => {
```

Comment:
I have a strongly typed class for this in my branch. Will add the nullfield options in my branch....